src/app/chat/create/page.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import { type ChangeEvent, useState } from 'react';
+import { Button, Header, Input } from '@/components/common';
+import styled from '@emotion/styled';
+
+const CreateChatPage = () => {
+  const [title, setTitle] = useState('');
+
+  const handleTitleInput = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleTitleChange = (e: ChangeEvent<HTMLInputElement>) => {
```

이렇게 쓰는건 어떨까용? 저는 뒤에 Change, Click이 붙는것이 좀 더 명시적으로 와닿더라구요 이것도 제안이기 때문에 정현님의 의견을 써주세요!